kolena/_utils/dataframes/validators.py

```diff
@@ -58,6 +58,11 @@ def validate_df_record_count(df: pd.DataFrame, max_records_allowed: Optional[int
 
 
 def _is_locator_cell_valid(cell: pa.typing.String) -> bool:
+    matcher = re.compile(r"^(s3|gs|http|https)://(.+/)+.+$")
```

Comment:
Separate question, should we disallow HTTP without TLS? Doesn't play nicely with modern browsers.

Reply:
I'm happy to sneak that in

Reply:
Done.